### ISSUE TITLE
Retry failed uploads

### ISSUE THREAD

**Issue by FunsKiTo** [Official-Husko/mul-tor#31]
**Is your feature request related to a problem? Please describe.**
At times, uploads aren't completed due to specific failures.

**Describe the solution you'd like**
Provide an option for failed downloads to be retried a specific number of times.

**Comment by Official-Husko** [comment]
What kind of failures? The only failures i have experienced until now are when the website was broken/down or the script was outdated due to website changes.

**Comment by FunsKiTo** [comment]
Yes, retry failed uploads for occasional hosting errors where the upload fails for some reason on their end.

**Comment by Official-Husko** [comment]
> Yes, retry failed uploads for occasional hosting errors where the upload fails for some reason on their end.
> 
> And attempt it X number of times, just in case the service happens to go down the moment you start uploading.

I will consider adding this. For now i have to still finish the other things.
Thanks for the Request.

**Comment by Official-Husko** [comment]
@FunsKiTo Just a quick question, do you use the proxies or random user agent function of this tool?

**Comment by FunsKiTo** [comment]
> @FunsKiTo Just a quick question, do you use the proxies or random user agent function of this tool?

Proxies.
